tests/StoringGroceries.tex

```diff
@@ -7,18 +7,22 @@ \subsection{Goal}
 \subsection{Focus}
 This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
 
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \begin{minipage}{0.70\textwidth}
 	\subsection{Setup}
 	\begin{enumerate}
-		\item \textbf{Location:} One of the bookcases or cupboards in the apartment is used for this test, one where a table is near or can be put. 
-		\item \textbf{Start position:} The robot will start between the cupboard and the table in a random orientation, but facing towards the Cupboard.
-		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects (See \ref{rule:scenario_objects}).
+		\item \textbf{Location:} This test can take place either inside or outside the arena. The testing area must have a bookcase or cupboard, and a newarby table. The maximum distance between the Table and the Cupboard is 2 meters.
+		\item \textbf{Start position:} The robot starts between the cupboard and the table in a random orientation, but facing towards the Cupboard.
+		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects grouped by category or likeliness (See \ref{rule:scenario_objects}). The cupboard has at least one free space for starting a new set.
```

Comment:
is this 30cm limit relevant? This indirectly excludes the lowest level of most cupboards. Can we always ensure to have 5 shelves?

Reply:
The 30 cm is relevant because I think is outside the specs of the HSR (Rules are cross-compatible with DSPL and OPL). Before DSPL it was 0-180m. 

Regarding the 5 shelves, so far there has been no problem.

Reply:
HSR can lift from the ground, with gripper and vacuum cup. It has a camera in the gripper so it should be even more capable than the average OPL in that regard.

Reply:
Done!